### ISSUE TITLE
My Works list includes element IDs which are not unique

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6781]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `Actions` drop down menu for each work in the `My Works` list contains element IDs which are not unique and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/dashboard/my/works?locale=en and run the Site Improve browser extension

![works_element_ids](https://github.com/samvera/hyrax/assets/32644254/d238dfff-6e30-4725-9fee-9a3f5c5497aa)

**Comment by aahurford** [comment]
Unable to replicate in pg or dev

**Comment by christhepianist** [comment]
Since this was still in Test and Confirm, I tested as well. I got the same results as Amanda. I also checked the HTML to confirm and there are no duplicate IDs in this page.
